src/app/services/theme/theme.service.ts

```diff
@@ -102,6 +104,37 @@ export class ThemeService {
     }
   }
 
+  private overrideBootstrapColors(color: string) {
```

Comment:
document function and write some some notes about the various blockers we have to deal with as bootstrap incorporates css variables (and which color function we're waiting for broad support for!)